openapi/data_object_service.swagger.yaml

```diff
@@ -71,11 +70,49 @@ paths:
           schema:
             $ref: '#/definitions/ErrorResponse'
       parameters:
-        - name: body
-          in: body
-          required: true
-          schema:
-            $ref: '#/definitions/ListDataBundlesRequest'
+        - name: alias
+          in: query
+          type: string
+          description: |-
+            OPTIONAL
+            If provided returns Data Bundles that have any alias that matches the
+            request.
+        - name: checksum
+          in: query
+          type: string
+          description: |-
+            REQUIRED
+            The hexlified checksum that one would like to match on.
+        - name: checksum_type
```

Comment:
This is a change in that ChecksumRequest was an object, but has been replaced by two parameters.